tests/integration/docs/nested/page1.md

```diff
@@ -27,6 +27,6 @@ But allows valid anchors such as
 
 ## Image Link absolute/relative
 
-<a href="../assets/hello-world.drawio.svg">![test](../assets/hello-world.drawio.svg)</a>
```

Comment:
This needed to be changed as it isn't a valid test with directory urls on. mkdocs won't change the link in the raw href to be correct. only those in markdown.